webapp/WEB-INF/conf/core_context.xml

```diff
@@ -50,7 +53,10 @@
     <bean id="siteMessageHandler" class="fr.paris.lutece.portal.service.message.SiteMessageHandler"/>
 
     <!--                      Search Engine                                   -->
-    <bean id="searchEngine" class="fr.paris.lutece.portal.service.search.LuceneSearchEngine" />
+    <bean id="searchEngine" class="fr.paris.lutece.portal.service.search.LuceneSearchEngine">
+        <aop:scoped-proxy proxy-target-class="false" />
```

Comment:
The ability to inject serializable proxies for singleton beans was added as part of Spring 3.0.0M4, issue https://jira.spring.io/browse/SPR-2121
The release note says (http://docs.spring.io/spring/docs/3.0.x/changelog.txt):
* scoped proxies are serializable now, for web scopes as well as for singleton beans